README.md

```diff
@@ -4,40 +4,28 @@ This project contains Kubernetes manifests for self-deployed OTLP ingest on Kube
 
 ## Running on GKE
 
-The recommended way to add the collector to your deployment is using `kubectl
-apply`. If running on a GKE Autopilot cluster (or any cluster with Workload
-Identity), you must follow the prerequisite steps to set up a Workload
-Identity-enabled service account below. Otherwise, you can skip to the next
-section.
-
-### Workload Identity prequisites
-
-Follow the [Workload Identity
-docs](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#authenticating_to)
-to set up an IAM service account in your GCP project with permission to use
-Workload Identity and write logs, traces, and metrics:
-
+Before we begin, set required environment variables:
 ```console
 export GCLOUD_PROJECT=<your project id>
+export PROJECT_NUMBER=<your project number>
 ```
 
-Then run the following to create the service account with the appropriate permissions:
+### Configure IAM Permissions
+
+Follow the [Workload Identity
+docs](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity)
+to allow the collector's kubernetes service account to write logs, traces, and metrics:
+
```

Comment:
Could we further clarify the instructions here: 

"Follow these docs to configure your application to use Workload Identity Federation for GKE and then ensure the following permissions are granted by running the following gcloud commands."

This will make sure that there are 2 steps involved -
1. Configuring Workload Identity (which this readme does not have instructions for, follow the GCP docs).
2. Adding bindings for the necessary permissions required to run this sample (you can copy-paste these commands).

Reply:
I don't think people will change whether or not workload identity is enabled in a cluster just to run this sample.

We can just tell people to skip the step if workload identity is disabled.

Reply:
updated